### PR TITLE
added favourite card html

### LINKED ISSUE #29: Create Cards for favourite pages
use card from countryCard in html
 - with; 
     + flag
     + country name 
     + button for remove from favourites 
image

### PR DESCRIPTION
**Pull request by Cratesy**
added mock card in favourite.html ready for js render 
 - added div favourite-card container for cards to be rendered into

### CODE REVIEW

File: favourites.html [hunk 1]
```diff
@@ -54,6 +54,17 @@ <h2 class="ui header">
         </div>
       </div>
     </header>
+    <div class="ui grid favourite-card">
+      <div class="ui centered card">
+        <div class="image">
+          <img src="https://restcountries.eu/data/esp.svg" />
+        </div>
+        <div class="content ui grid col-4">
+          <a class="header">Spain</a>
```
Comment (natasha-mann) [inline favourites.html]: Can you center the header? I think it's center aligned or something :)

Comment (natasha-mann) [reply favourites.html]: "ui center aligned header"

Comment (Cratesy) [reply favourites.html]: centered header for you =P

**Review by natasha-mann — APPROVED** [review]
Great :)

**Pull request merged**